src/DemuxTopic.cpp

```diff
@@ -23,19 +23,19 @@ ProcessMessageResult
 DemuxTopic::process_message(FlatbufferMessage const &Message) {
   Logger->trace("Message received from: {}", Message.getSourceName());
   try {
-    auto &CurrentSource = TopicSources.at(Message.getSourceName());
+    auto &CurrentSource = TopicSources.at(Message.getSourceHash());
     auto ProcessingResult = CurrentSource.process_message(Message);
     ++messages_processed;
     return ProcessingResult;
   } catch (std::out_of_range &e) {
-    Logger->trace("Source with name \"{}\" is not in list.",
-                  Message.getSourceName());
+    Logger->trace("Source with name \"{}\" and ID \"{}\" is not in list.",
```

Comment:
Braces don't need to be escaped.

Reply:
I don't understand, I am not escaping braces here. I am escaping the quotation marks.

Reply:
Oops, somehow missed that ^^

Reply:
np